src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery firstBall = new Ball();
+        Lottery secondBall = new Ball();
+        Lottery thirdBall = new Ball();
+
+        Lottery[] lotteries = new Lottery[]{firstBall, secondBall, thirdBall};
+        for (Lottery lottery : lotteries) {
+            System.out.println(lottery.getRandomBall());
+        }
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
The Ball should not extend Lottery. Those are absolutely different objects.
Lottery should have a method that creates a random ball